internal/adapters/events_forwarder/noop_forwarder/noop_forwarder.go

```diff
@@ -24,27 +24,80 @@ package noop_forwarder
+
+	pb "github.com/topfreegames/protos/maestro/grpc/generated"
 )
 
-type noopForwarder struct{}
+type noopForwarder struct {
+	forwarderGrpc ports.ForwarderGrpc
+	GrpcClient    pb.GRPCForwarderClient
+}
 
-func NewNoopForwarder() *noopForwarder {
-	return &noopForwarder{}
+func NewNoopForwarder(forwarderGrpc ports.ForwarderGrpc) *noopForwarder {
+	return &noopForwarder{
+		forwarderGrpc: forwarderGrpc,
+	}
 }
 
 // ForwardRoomEvent forwards room events. It receives the room event attributes and forwarder configuration.
-func (f *noopForwarder) ForwardRoomEvent(ctx context.Context, eventAttributes events.RoomEventAttributes, forwarder forwarder.Forwarder) error {
-	return nil
+func (f *noopForwarder) ForwardRoomEvent(ctx context.Context, eventAttributes events.RoomEventAttributes, forwarder entities.Forwarder) error {
+	if eventAttributes.EventType == events.Arbitrary {
+		event := pb.RoomEvent{
+			Room: &pb.Room{
+				Game:     eventAttributes.Game,
+				RoomId:   eventAttributes.RoomId,
+				Host:     eventAttributes.Host,
+				Port:     eventAttributes.Port,
+				Metadata: *fromMapInterfaceToMapString(eventAttributes.Attributes),
+			},
+			EventType: entities.FromForwardTypeToString(forwarder.ForwardType),
+			Metadata:  *fromMapInterfaceToMapString(forwarder.Options.Metadata),
+		}
+
+		eventResponse, err := f.forwarderGrpc.SendRoomEvent(ctx, forwarder, &event)
+		return handlerGrpcClientResponse(forwarder, eventResponse, err)
+	}
+
+	if eventAttributes.EventType == events.Ping {
+		event := pb.RoomStatus{
+			Room: &pb.Room{
+				Game:     eventAttributes.Game,
+				RoomId:   eventAttributes.RoomId,
+				Host:     eventAttributes.Host,
+				Port:     eventAttributes.Port,
+				Metadata: *fromMapInterfaceToMapString(eventAttributes.Attributes),
+			},
+			StatusType: fromRoomPingEventTypeToRoomStatusType(*eventAttributes.PingType),
+		}
+
+		eventResponse, err := f.forwarderGrpc.SendRoomReSync(ctx, forwarder, &event)
+		return handlerGrpcClientResponse(forwarder, eventResponse, err)
+	}
+
+	return errors.NewErrUnexpected("failed to forwarder event room. event type doesn't exists \"%s\"", eventAttributes.EventType)
 }
 
 // ForwardPlayerEvent forwards a player events. It receives the player events attributes and forwarder configuration.
-func (f *noopForwarder) ForwardPlayerEvent(ctx context.Context, eventAttributes events.PlayerEventAttributes, forwarder forwarder.Forwarder) error {
-	return nil
+func (f *noopForwarder) ForwardPlayerEvent(ctx context.Context, eventAttributes events.PlayerEventAttributes, forwarder entities.Forwarder) error {
+	event := pb.PlayerEvent{
+		PlayerId: eventAttributes.PlayerId,
+		Room: &pb.Room{
+			RoomId:   eventAttributes.RoomId,
+			Metadata: *fromMapInterfaceToMapString(eventAttributes.Other),
+		},
+		EventType: fromPlayerEventTypeToGrpcPlayerEventType(eventAttributes.EventType),
+		Metadata:  *fromMapInterfaceToMapString(forwarder.Options.Metadata),
```

Comment:
@lyracampos this metadata should be a merge between `forwarder.Options.Metadata` and `eventAttributes.Other`, you can check the logic in v9 `mergePlayerInfos` method (plugins/grpc/forwarder.go:362)